app/views/fields/nested_has_many/_fields.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="nested-fields" style="width:100%">
+
+  <%# TODO filter out the parent relationship, without this crude array slicing %>
+  <% field.associated_form.attributes[1..-1].each do |attribute| -%>
+    <div class="field-unit field-unit--<%= attribute.html_class %>">
```

Comment:
It gives you the type of field that is being rendered. For example, `Administrate::Field::String` attributes would return `"string"`. It's useful for styling fields based on the type of attribute.